### DISCUSSION TITLE
Does not connect to running instance of `wezterm-mux-server`

### DISCUSSION THREAD

**Discussion by riley-martin** [wezterm/wezterm#3085] [Q&A]
I have a server that I installed wezterm on so I could use its mupltiplexing capabilities without the overhead of something like tmux. The connection works fine, but if I detach the pane and try to reconnect, it starts a new server instance instead. If I ssh into the server after detaching, the `/run/user/1000/wezterm` directory is not present, but `ps -aux | grep wez` shows multiple server instances (if I connected more than once anyway). Strangely, though, if I have an ssh session open before I detach the wezterm session, and leave it open until I connect wezterm again, everything works as expected.  

Wezterm version: 20230208-091928-fd5c6dce

I am running Artix Linux (on both machines) which does not use systemd, maybe this has something to do with it?

**Comment by wez** [comment]
My guess is that the remote host is using systemd and is configured to kill off any of the processes from the originating ssh session when that disconnects.

You may want to configure a systemd unit of your own that launches `wezterm-mux-server` independently from your login session.

**Reply by riley-martin** [reply]
The host is not using systemd, its just my home server, and it is running Artix with openrc.  

> and is configured to kill off any of the processes from the originating ssh session when that disconnects.

This could still be the problem, though. I'll see what I can figure out.
